modular_skyrat/modules/modular_items/code/necklace.dm

```diff
@@ -12,6 +12,7 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/neck.dmi'
 	icon_state = "ashnecklace"
 	w_class = WEIGHT_CLASS_SMALL //allows this to fit inside of pockets.
+	resistance_flags = FIRE_PROOF
```

Comment:
Needs modularization. Just make an override folder in modular_zubber and have this bit of code in it.

```
/obj/item/clothing/neck/necklace/ashwalker
	resistance_flags = FIRE_PROOF
```

Reply:
Or alternatively PR it upstream

Reply:
That's done 👍